apps/frontend/src/libs/modules/store/libs/middlewares/error-handling.middleware.ts

```diff
@@ -0,0 +1,17 @@
+import { isRejectedWithValue, type Middleware } from "@reduxjs/toolkit";
+import { toast } from "react-toastify";
+
+import { type AsyncThunkRejectValue } from "~/libs/types/types.js";
+
+const middleware: Middleware = () => {
+	return (next) => (action) => {
+		if (isRejectedWithValue(action)) {
+			const rejectedValue = action.payload as AsyncThunkRejectValue;
+			toast.error(rejectedValue.message);
```

Comment:
create a new package-wrapper for the notification service

Reply:
Ok, great idea